po/es.po

```diff
@@ -319,12 +319,14 @@ msgid "Header Text"
 msgstr "Texto de encabezado"
 
 #: oomox_gui/theme_model.py:154
+<<<<<<< HEAD
+=======
 #, fuzzy
+>>>>>>> 0eed3cb424cfd3e09d78c9c7abe0cf7c8ca63a56
```

Comment:
this conflict not resolved

Reply:
and you can just search onwards for all "===" text

Reply:
I will back up the PO files, will remove the fork and the repository and refork the repository, OK?

Reply:
yup, that's how i was resolving merge conflicts before i've learned mergetools :-)

i don't have any objection about which way you'll do, though familiarity with merge-sidecases could be useful for you in the future